### PR TITLE
New package: NetEase.CloudMusicXStudio version 1

### PR DESCRIPTION
**Pull request by SpecterShell**
- [X] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [X] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [X] This PR only modifies one (1) manifest
- [X] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [X] Have you tested your manifest locally with `winget install --manifest <path>`?
- [X] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

 ###### Microsoft Reviewers: codeflow:open?pullrequest=https://github.com/microsoft/winget-pkgs/pull/110012&drop=dogfoodAlpha

### CODE REVIEW

**Comment by SpecterShell** [comment]
The URL is not encoded during the pipeline's Installer Scan phase, which may result in 400 Bad Request for some URLs. @denelon 
https://dev.azure.com/ms/winget-pkgs/_build/results?buildId=468362&view=logs&jobId=871b4eed-22de-5cb6-c1c8-82f190de633f&j=871b4eed-22de-5cb6-c1c8-82f190de633f&t=523ceef4-ddf7-55a0-c83d-dd0f77b67744
@Trenly I don't think the URLs can be decoded at present. 🤔

**Review by ItzLevvie — APPROVED** [review]

**Comment by Trenly** [comment]
> The decoded URL is not encoded during the pipeline's Installer Scan phase, which may result in 400 Bad Request for some URLs. @denelon [dev.azure.com/ms/winget-pkgs/_build/results?buildId=468362&view=logs&jobId=871b4eed-22de-5cb6-c1c8-82f190de633f&j=871b4eed-22de-5cb6-c1c8-82f190de633f&t=523ceef4-ddf7-55a0-c83d-dd0f77b67744](https://dev.azure.com/ms/winget-pkgs/_build/results?buildId=468362&view=logs&jobId=871b4eed-22de-5cb6-c1c8-82f190de633f&j=871b4eed-22de-5cb6-c1c8-82f190de633f&t=523ceef4-ddf7-55a0-c83d-dd0f77b67744) @Trenly I don't think the URLs can be decoded at present. 🤔

Thats very odd

**Comment by stephengillie** [comment]
Manual Validation failed with:
> 2:29:09 PM:  > 2023-06-19 14:29:08.705 [CLI ] Package hash verification failed. SHA256 in manifest [be10c7d0f6a2ce8b79c9fb43b8844e638a06c21d67640883829eadfb74168b34] does not match download [efa9424a5a495f39853ca483fe24c1cacd93318974150875d9740b33bd0d38d2]

File: manifests/n/NetEase/CloudMusicXStudio/1/NetEase.CloudMusicXStudio.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+Scope: machine
+UpgradeBehavior: install
+FileExtensions:
+- myp
+- svip
+- svip3
+Installers:
+- Architecture: x64
+  InstallerUrl: https://xcloudsingblob.blob.core.chinacloudapi.cn/xstudio-public/production/18/%E7%BD%91%E6%98%93%E4%BA%91%E9%9F%B3%E4%B9%90%C2%B7XStudio_1.0.0_Windows_x86_64.exe
+  InstallerSha256: BE10C7D0F6A2CE8B79C9FB43B8844E638A06C21D67640883829EADFB74168B34
```
Comment (stephengillie) [inline manifests/n/NetEase/CloudMusicXStudio/1/NetEase.CloudMusicXStudio.installer.yaml:16]:
```suggestion
  InstallerSha256: EFA9424A5A495F39853CA483FE24C1CACD93318974150875D9740B33BD0D38D2
```

**Review by stephengillie — APPROVED** [review]

**Pull request merged**